### PR TITLE
02.Git.Local_Maxim_Radaman

### PR DESCRIPTION
**Pull request by redman27**
02.Git.Local_Maxim_Radaman

### CODE REVIEW

**Review by pluhin — CHANGES_REQUESTED** [review]
Please leave only report in your PR

**Comment by redman27** [comment]
files removed, also change syntax in report file

File: Maxim_Radaman/02.Git.Local/02.GIT.Local.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  273  ll
+  274  git checkout features/do_one 
+  275  git rebase hotfix/we_gonna_die 
+  276  ll
+  277  git checkout master
+  278  history > 02.GIT.Local.md
+```
+# Master commits:
+```bash
+<<<<<<< HEAD
```
Comment (pluhin) [inline Maxim_Radaman/02.Git.Local/02.GIT.Local.md:66]: @redman27 

Looks like conflict?

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Maxim_Radaman/02.Git.Local/02.GIT.Local.md (hunk 1 above)
Comment (redman27) [reply Maxim_Radaman/02.Git.Local/02.GIT.Local.md]: fixed

**Review by pluhin — APPROVED** [review]
Good, approved

**Pull request merged**